### PR TITLE
Fix/toggles tidy

### PR DESCRIPTION
**Pull request by jamesgorrie**
I was about to add one, but thought it worth removing some first.

### CODE REVIEW

File: toggles/webapp/toggles.ts [hunk 1]
```diff
@@ -54,11 +40,5 @@ export default {
       description:
         'Displays body copy in Helvetica regular (where it is currently Helvetica light)',
     },
```
Comment (jamesgorrie) [inline toggles/webapp/toggles.ts:42]: @gestchild do we still need `openWithAdvisoryPrototype`? Is it there as a protection, or was it there for us to test?

File: catalogue/webapp/pages/_app.tsx [hunk 2]
```diff
@@ -1,9 +1,5 @@
 import { NextWebVitalsMetric, AppProps } from 'next/app';
 import App from '@weco/common/views/pages/_app';
-// this css should be included in the ColorPicker component
```
Comment (jamieparkinson) [inline catalogue/webapp/pages/_app.tsx:3]: 🎉

**Review by jamieparkinson — APPROVED** [review]

File: toggles/webapp/toggles.ts (hunk 1 above)
Comment (gestchild) [reply toggles/webapp/toggles.ts:42]: it was left there as a protection, but probably safe to remove now

Comment (jamesgorrie) [reply toggles/webapp/toggles.ts:42]: Is it reliant on another service? And if so, do we handle failures gracefully?

Comment (jamesgorrie) [reply toggles/webapp/toggles.ts:42]: Maybe we can chat about this in another PR

**Pull request merged**